webui/react/src/hooks/useModal/UserSettings/useModalCreateGroup.tsx

```diff
@@ -129,9 +129,9 @@ const ModalForm: React.FC<Props> = ({ form, users, group, groupRoles }) => {
               optionFilterProp="children"
               placeholder={'Add Roles'}
               showSearch>
-              {Loadable.match(knowRolesLoadable, {
+              {Loadable.match(rolesLoadable, {
                 Loaded: () =>
```

Comment:
Not introduced in this PR but the `Loaded` case (here and elsewhere) should be unpacking the roles here instead of the `getOrElse` above.

Reply:
Thanks